sql/moz-fx-data-shared-prod/firefox_ios_derived/profile_dau_metrics_marketing_geo_testing_v1/query.sql

```diff
@@ -1,6 +1,7 @@
 SELECT
   active_users.submission_date,
   active_users.normalized_channel,
+  active_users.channel,
```

Comment:
@SuYoungHong do we want to rename `normalized_channel` to `channel` in this table?

Reply:
I'm okay with whatever convention is decided, as long as it's consistent through the ETL (although if we used normalized, probably good idea to make sure its' gone through the normalization process)